src/lib/mousetrap-record.js

```diff
@@ -0,0 +1,225 @@
+/**
+ * This extension allows you to record a sequence using Mousetrap.
```

Comment:
We could move it to app/ and exclude in gulp or fix all annoying jslint errors. If the library is complete garbage we could use this: https://dmauro.github.io/Keypress/
But it's a much bigger library in comparison to this small Mousetrap plugin.